src/platforms/java/config.yml

```diff
@@ -2,3 +2,6 @@ title: Java
 caseStyle: camelCase
 supportLevel: production
 sdk: sentry.java
+categories:
+  - desktop
+  - server
```

Comment:
we could also add `serverless` but we don't have official docs about it yet, same for Dart, so I guess its not a good idea.

Reply:
I wouldn't add serverless then.